docs/docs/getting-started/04-examples/02-editor-example.md

```diff
@@ -15,88 +15,91 @@ on the page to expand either the code editor or graph view to full screen.
 
-Repeat for a second button to show manual deactivation / activation
+## Adding and deleting elements
 
-Then say: now that we learned to add buttons and create event edges, try to do the same for components. As an exercise,
-search for and add timer components, set thea auto-lifecycle events, and connect the transition edges to recreate the
-example from scratch.
--->
+Let's now make the example a bit more interesting by adding more elements to the graph using the sidebar menu; press the
+(+) button in the corner of the graph to open the sidebar. At the top, you will see special entries
+"Hardware Interface", "Trigger Events Button", "Sequence" and "Condition", which will all have a dedicated example later
+on. These are followed by a list of components with a name and brief description.
+
+Trigger Events Buttons are interactive elements that can be used to trigger events through mouse clicks and interact
+with the flow of the application. Click on the Trigger Events Button in the sidebar to add such an element to the
+application. A new button should appear on the graph and in the code.
+
+The sidebar menu can be closed by clicking anywhere in the graph. Click and drag on the button to change its position in
+the graph. This will also update the associated position in the code editor.
+
+To delete an element from the graph, press the small menu icon in its top right corner, then click Remove.
 
 ## Creating and deleting event edges
 
-The timers are connected with a Transition event edge. To change the event type on an edge, click on the label. This
-will open a selection menu showing other available event types (for example, Load, Unload, Configure...). Choosing
+The timers are already connected with a Transition event edge. To change the event type on an edge, click on the label.
+This will open a selection menu showing other available event types (for example, Load, Unload, Configure...). Choosing
 a different event type will close the selection menu and update the YAML code accordingly.
 
-![timer example](./assets/event-edge.png)
+![event edge](./assets/event-edge.png)
 
 Clicking on an edge also selects it, which is indicated by the increased line thickness. Clicking away from the edge
 will deselect it. While the edge is selected, press the delete key to delete the edge. This will also remove the
 event from the YAML representation.
 
-To create a new edge, move the mouse over the event source handle next to a component predicate until the cursor changes
-to a targeting reticule. Then, click and drag to create a draft edge. Pull it towards a target handle (on the top left
-of a component) until the cursor changes again and the draft edge snaps in place, then let go of the mouse button.
-
-If the connection is valid, it will create a new event edge with the default event (Load).
+To create a new edge, move the mouse over the event source handle until the cursor changes to a targeting reticule.
```

Comment:
```suggestion
To create a new edge, move your mouse over the event source handle until the cursor changes to a targeting reticle.
```

perhaps both are valid :thinking:

Reply:
I think both are :+1: It's not in my vocabulary and I took the Docs version :smile: